src/crashreporter/crashreporterapp.cpp

```diff
@@ -11,10 +11,6 @@
 #include "crashreporterfactory.h"
 
 int CrashReporterApp::main(int argc, char* argv[]) {
-#if QT_VERSION < 0x060000
-  // This flag is set by default in qt6.
-  QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
-#endif
```

Comment:
Unrelated?

Reply:
Yes, but since we are all in on Qt6, I thought I'd pull it.